sources/packages/api/src-sql/Student/Create-student.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IF NOT EXISTS students(
+  id SERIAL PRIMARY KEY,
+  contact_info JSONB NOT NULL,
+  sin VARCHAR(10) NOT NULL,
+
+  -- Reference Columns
+  user_id INT NULL DEFAULT NULL REFERENCES users(id) ON DELETE CASCADE,
```

Comment:
I believe that would be good to not allow NULL for user_id because a student should not exist without it.

Reply:
FIXED